frontend/pages/admin/UserManagementPage/UserManagementPage.tsx

```diff
@@ -325,42 +331,6 @@ const UserManagementPage = ({ router }: IUserManagementProps): JSX.Element => {
       );
     }
 
-    if (currentUser?.id === userEditing.id) {
```

Comment:
No longer used since if the current user clicks edit to themselves, they are now brought to the UserSettingsPage instead of the EditUserModal